src/main/java/org/broadinstitute/hellbender/tools/copynumber/DetermineGermlineContigPloidy.java

```diff
@@ -28,73 +28,113 @@
- *      only contain the "-calls" subdirectory.
+ *     <dd>If a directory containing previously inferred ploidy model parameters is provided via the
+ *     <code>--model &lt;path&gt;</code> argument, then the tool will run in the CASE mode. In this mode, the
+ *     parameters of the ploidy model are loaded from the provided directory and only sample-specific quantities are
+ *     inferred. Subsequently, the output directory will only contain the "-calls" subdirectory.
  *
  *      <p>In the CASE mode, the contig ploidy prior table is taken directly from the provided model parameters
  *      path and must be not provided again.</p></dd>
  * </dl>
  *
+ * <h3>Important Remarks</h3>
+ * <dl>
+ *     <dt>Choice of hyperparameters:</dt>
+ *     <dd><p>The quality of ploidy model parametrization and the sensitivity/precision of germline karyotyping are
+ *     sensitive to the choice of model hyperparameters, including standard deviation of mean contig coverage bias
+ *     (set by <code>--mean-bias-standard-deviation &lt;value&gt;</code>), mapping error rate
+ *     (set by <code>--mapping-error-rate &lt;value&gt;</code>), and the typical scale of contig- and sample-specific
+ *     unexplained variance (set by <code>--global-psi-scale &lt;value&gt;</code> and <code>--sample-psi-scale
+ *     &lt;value&gt;</code>, respectively). It is crucial to note that these hyperparameters are <em>not</em> universal
+ *     and must be tuned for each sequencing protocol and properly set at runtime.</p></dd>
+ *
+ *     <dt>Mosaicism and fractional ploidies:</dt>
+ *     <dd><p>The model underlying this tool assumes integer ploidy states (in contrast to fractional/variable ploidy
+ *     states). Therefore, it is to be used strictly on germline samples and for the purpose of sex determination,
+ *     autosomal aneuploidy detection, or as a part of GATK germline CNV calling pipeline. The presence of large somatic
+ *     events and mosaicism (e.g. sex chromosome loss and somatic trisomy) will naturally lead to unreliable
+ *     results. We strongly recommended inspecting genotyping qualities (GQ) from the tool output and considering to drop
+ *     low-GQ contigs in downstream analyses. Finally, given the Bayesian status of this tool, we suggest including as many
+ *     high-quality germline samples as possible for ploidy model parametrizaton in the COHORT mode. This will downplay
+ *     the role of questionable samples and will yield a more reliable estimation of genuine sequencing biases.</p></dd>
+ *
+ *     <dt>Coverage-based germline karyotyping:</dt>
+ *     <dd><p>Accurate germline karyotyping requires incorporating SNP allele frequencies and counts data in a
+ *     unified probabilistic model and is beyond the scope of the present tool. The current implementation only uses
+ *     counts data for karyotyping and while being fast, it may not provide the most reliable results.</p></dd>
+ *
```

Comment:
Not in this diff, but remove periods from the end of `summary` and `oneLineSummary` here and in GermlineCNVCaller.

Reply:
Done.